api/middleware/rateLimiter/index.js

```diff
@@ -0,0 +1,33 @@
+import rateLimit from 'express-rate-limit';
+
+const isProduction = process.env.NODE_ENV === 'production';
+
+const API_RATE_LIMIT_WINDOW_MS = Number(process.env.API_RATE_LIMIT_WINDOW_MS) || 15 * 60 * 1000;
+const API_RATE_LIMIT_MAX_PROD = Number(process.env.API_RATE_LIMIT_MAX_PROD) || 300;
+
+const rateLimiter = rateLimit({
+    windowMs: API_RATE_LIMIT_WINDOW_MS,
+    max: API_RATE_LIMIT_MAX_PROD,
```

Comment:
should this be "limit"?
https://express-rate-limit.mintlify.app/reference/configuration#limit

Can't se "max" in the docs